src/scenarios/mauiandroid/pre.py

```diff
@@ -3,47 +3,58 @@
 '''
 import sys
 import os
+import requests
+import subprocess
 from zipfile import ZipFile
 from performance.logger import setup_loggers, getLogger
 from shutil import copyfile
+from shared import const
 from shared.precommands import PreCommands
-from shared.const import PUBDIR
 from argparse import ArgumentParser
+from test import EXENAME, MAUIVERSIONFILE
 
 setup_loggers(True)
 
-parser = ArgumentParser()
-parser.add_argument('--unzip', help='Unzip APK and report extracted tree', action='store_true', default=False)
+parser = ArgumentParser(add_help=False)
 parser.add_argument(
-        '--apk-name',
-        dest='apk',
-        required=True,
+        '-o', '--output',
+        dest='output_dir',
+        required=False,
         type=str,
-        help='Name of the APK to setup')
-args = parser.parse_args()
-
-if not os.path.exists(PUBDIR):
-    os.mkdir(PUBDIR)
-apkname = args.apk
-apknamezip = '%s.zip' % (apkname)
-if not os.path.exists(apkname):
-    getLogger().error('Cannot find %s' % (apkname))
-    exit(-1)
-if args.unzip:
-    if not os.path.exists(apknamezip):
-        copyfile(apkname, apknamezip)
-
-    with ZipFile(apknamezip) as zip:
-        zip.extractall(os.path.join('.', PUBDIR))
-
-    assets_dir = os.path.join(PUBDIR, 'assets')
-    assets_zip = os.path.join(assets_dir, 'assets.zip')
-    with ZipFile(assets_zip) as zip:
-        zip.extractall(assets_dir)
-
-    os.remove(assets_zip)
-else:
-    copyfile(apkname, os.path.join(PUBDIR, apkname))
-
-
-
+        help='capture of the output directory')
+args, unknown_args = parser.parse_known_args()
+
+# Download what we need
+with open ("MauiNuGet.config", "wb") as f:
+    f.write(requests.get(f'https://raw.githubusercontent.com/dotnet/maui/net7.0/NuGet.config', allow_redirects=True).content)
```

Comment:
I suppose this will get (forward)ported to `main` eventually and hence the `net7.0` here (and below) should come from variable.

Reply:
This has been fixed